test/data/Datatypes23/Datatypes23Attr_textLimitedTooLong.xtf

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TRANSFER xmlns="http://www.interlis.ch/INTERLIS2.3">
+	<HEADERSECTION SENDER="ili2fgdb-3.10.0-20170630" VERSION="2.3">
+		<MODELS>
+			<MODEL NAME="Datatypes23" VERSION="2016-12-21" URI="mailto:ce@eisenhutinformatik.ch"></MODEL>
+		</MODELS>
+	</HEADERSECTION>
+	<DATASECTION>
+		<Datatypes23.Topic BID="b1">
+			<Datatypes23.Topic.ClassAttr TID="ClassAttr.1">
+				<aufzaehlung>mehr.vier</aufzaehlung>
+				<aI32id OID="22"></aI32id>
+				<aUuid OID="15B6BCCE-8772-4595-BF82-F727A665FBF3"></aUuid>
+				<aStandardid OID="chgAAAAAAAAA0azD"></aStandardid>
+				<textLimited>1111111111111111111111111111111111111111</textLimited>
```

Comment:
Die Option betrifft ja nicht nur zu lange Text-Attribute, sondern alle simple-Type Attribute, also Test erweitern, so dass z.B. auch für die Aufzählung Werte importiert werden, die nicht in der Aufzählung vorkommen, oder Datumswerte, die keine Datum sind, oder Text bei Zahlen-Attribute usw.